SignalServiceKit/src/Protos/Generated/SSKProto.swift

```diff
@@ -0,0 +1,3588 @@
+            contact.append(wrapped)
+        }
+
+        // MARK: - Begin Validation Logic for SSKProtoDataMessage -
+
+        // MARK: - End Validation Logic for SSKProtoDataMessage -
+
+        let result = SSKProtoDataMessage(proto: proto,
+                                         attachments: attachments,
+                                         group: group,
+                                         quote: quote,
+                                         contact: contact)
+        return result
+    }
+}
+
+// MARK: - SSKProtoNullMessage
+
+@objc public class SSKProtoNullMessage: NSObject {
+
+    // MARK: - SSKProtoNullMessageBuilder
+
+    @objc public class SSKProtoNullMessageBuilder: NSObject {
+
+        private var proto = SignalServiceProtos_NullMessage()
+
+        @objc public override init() {}
+
+        @objc public func setPadding(_ valueParam: Data) {
+            proto.padding = valueParam
+        }
+
+        @objc public func build() throws -> SSKProtoNullMessage {
+            let wrapper = try SSKProtoNullMessage.parseProto(proto)
+            return wrapper
+        }
+    }
+
+    fileprivate let proto: SignalServiceProtos_NullMessage
+
+    @objc public var padding: Data? {
+        guard proto.hasPadding else {
+            return nil
+        }
+        return proto.padding
+    }
+    @objc public var hasPadding: Bool {
+        return proto.hasPadding
+    }
+
+    private init(proto: SignalServiceProtos_NullMessage) {
+        self.proto = proto
+    }
+
+    @objc
+    public func serializedData() throws -> Data {
+        return try self.proto.serializedData()
+    }
+
+    @objc public class func parseData(_ serializedData: Data) throws -> SSKProtoNullMessage {
+        let proto = try SignalServiceProtos_NullMessage(serializedData: serializedData)
+        return try parseProto(proto)
+    }
+
+    fileprivate class func parseProto(_ proto: SignalServiceProtos_NullMessage) throws -> SSKProtoNullMessage {
+        // MARK: - Begin Validation Logic for SSKProtoNullMessage -
+
+        // MARK: - End Validation Logic for SSKProtoNullMessage -
+
+        let result = SSKProtoNullMessage(proto: proto)
+        return result
+    }
+}
+
+// MARK: - SSKProtoReceiptMessage
+
+@objc public class SSKProtoReceiptMessage: NSObject {
+
+    // MARK: - SSKProtoReceiptMessageType
+
+    @objc public enum SSKProtoReceiptMessageType: Int32 {
+        case delivery = 0
+        case read = 1
+    }
+
+    private class func SSKProtoReceiptMessageTypeWrap(_ value: SignalServiceProtos_ReceiptMessage.TypeEnum) -> SSKProtoReceiptMessageType {
+        switch value {
+        case .delivery: return .delivery
+        case .read: return .read
+        }
+    }
+
+    private class func SSKProtoReceiptMessageTypeUnwrap(_ value: SSKProtoReceiptMessageType) -> SignalServiceProtos_ReceiptMessage.TypeEnum {
+        switch value {
+        case .delivery: return .delivery
+  
```

Comment:
Indentation seems to have gone awry at some point

Reply:
Fixed.